java/src/main/java/org/extism/sdk/LibExtism.java

```diff
@@ -0,0 +1,32 @@
+package org.extism.sdk;
+
+import com.sun.jna.Library;
+import com.sun.jna.Native;
+import com.sun.jna.Pointer;
+import com.sun.jna.ptr.IntByReference;
+//import com.sun.jna.Platform;
+
+public interface LibExtism extends Library {
+    LibExtism INSTANCE = (LibExtism) Native.loadLibrary("/usr/local/lib/libextism.dylib", LibExtism.class);
```

Comment:
There is probably a standard way to do this that is platform independent. Might be some code in JNA which does it or JNI. Will search it out.